### DISCUSSION TITLE
v2/md-enhance/zh/guide/container

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#421] [Announcements]
# v2/md-enhance/zh/guide/container

插件可以为你添加提示、注释、信息、注意、警告和详情自定义容器的支持。 配置 tab TS tab JS 演示 信息容器。 注释容器。 提示容器 警告容器 危险容器 详情容器 自定义标题 一个有 代码 和 链接 (#演示) 的信息容器。 自定义标题 一个有 代码 和 链接 (#演示) 的注释容器。 自定义标题 一个有 代码 和 链接 (#演示) 的提示容...

https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/container.html

**Comment by nwuzmedoutlook** [comment]
不支持卡片列表格式的吗😐

**Comment by nwuzmedoutlook** [comment]
类似这种的没看到例子，试了几种代码都不生效呢
![2023-01-11_121326](https://user-images.githubusercontent.com/78712654/211716253-a004e73d-9e5f-4c46-9745-2b937925e43c.png)
